pinc/iso_lang_list.inc

```diff
@@ -1,4 +1,61 @@
 <?php
+// This file deals with language names and codes that are used with projects.
+// This includes languages assigned to projects and mapping those languages to
+// system dictionaries used with WordCheck. The functions in language.inc
+// deal with localizations/translations and system locales.
+
+// Returns ISO 639 code(s) for language(s) of a project.
+// $which could be "primary", "secondary" or "all":
+// - "primary" returns the code of primary language
+// - "secondary" returns the code of secondary language, or NULL if the
+//   secondary language doesn't exist
+// - "all" returns array with the code(s) of both language(s); if the secondary
+//   language doesn't exist, the array only has one element
+// If a language doesn't have ISO 639 code, NULL is returned; this might change
+function proj_lang_code($proj_lang, $which="all")
```

Comment:
an enum would be nice to specifically enumerate options

Reply:
PHP doesn't have enums unfortunately.

Interestingly, the only value that is ever used when calling this function is `primary` which tells me we might just want to remove the option entirely and simplify the function:
```
$ grep -R proj_lang_code *
pinc/wordcheck_engine.inc:        $langcode = proj_lang_code($language, "primary");
pinc/wordcheck_engine.inc:        $langcode = proj_lang_code($language, "primary");
pinc/LPage.inc:        $lang = proj_lang_code($project->language,"primary");
```

Reply:
Let's leave this as-is for this MR. I see how I want to rework project languages slightly differently but think that needs to happen in a follow-up.